artemis-web/src/main/java/org/apache/activemq/artemis/component/AuthenticationFilter.java

```diff
@@ -43,14 +43,21 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
       filterChain.doFilter(servletRequest, servletResponse);
       if (AuditLogger.isAnyLoggingEnabled()) {
          int status = ((Response) servletResponse).getStatus();
-         //status 200 means that the user has been authenticated, anything else must be a failure
-         if (status == 200) {
+         if (status < 300) {
```

Comment:
Does 100 continue need to be handled as well - would that ever reach here?

Reply:
Status ranges updated